foundation/foundation-upgrade/src/test/resources/specifications/v1_7_0/enable-habushu-build-cache-migration.feature

```diff
@@ -0,0 +1,17 @@
+@habushu-build-cache-migration
+Feature: Adds the <build><directory>dist</directory></build> tag to the pom file if not present.
+
+  Scenario: Migrate a pom file
+    Given a habushu-packaged pom file where the build directory is unspecified
+    When the migration executes
+    Then the pom file contains a specification of the build directory
+
+  Scenario: A pom file is not packaged by habushu
```

Comment:
S: Only migrates pom files that are habushu packaged... (or similar)

Reply:
Done.